CHANGELOG.md

```diff
@@ -43,6 +43,7 @@
 - PR #447 Update Python README with info about DeviceBuffer/MemoryResource and external libraries
 - PR #456 Minor cleanup: always use rmm/-prefixed includes
 - PR #461 cmake improvements to be more target-based
+- PR #462 export cmake config for downstream use
```

Comment:
changelog entry needs to be moved to 0.16 section

Reply:
I guess I'm somewhat confused as to why you're branching off `branch-0.16` now, while `branch-0.15` currently undergoes what looks like major changes to me? Unless you have urgent things that need to be into 0.16 now, wouldn't it be easier to wait until the release -- otherwise you'll have to  merge 0.15 into 0.16 repeatedly and end up with a less than pretty history?

In any case, I did merge `branch-0.16`, but I also put this back into [WIP] because it seems to me it'd be better to do this over in 0.16, actually starting with sorting out thirdparty dependencies first and then doing the cmake exports based on that, without requiring, e.g., the spdlog workaround mentioned above. But I'll do whatever you say the process should be.

Reply:
There's an automerger that runs which will automatically forward port things merged into branch-0.15 into branch-0.16. The major changes going on still in branch-0.15 have been discussed for the past ~6 months where there's a higher comfort level with the changes being made (mainly deprecation of cnmem).

Sorting out the thirdparty dependencies sounds great. @mt-jones and @millerhooks have been exploring / tinkering in this area as well so adding them to the conversation 😄

Reply:
automerger -- fancy ;) Cool, that'll do.

Reply:
Lots of useful docs here: https://docs.rapids.ai/maintainers  It's pretty great to have an awesome devops team. :)